OfficeAddin/05 Using modern JavaScript/Demos/02 Office Add-in using Angular/README.md

```diff
@@ -1,13 +1,12 @@
 # Office Add-ins: Build an Office Add-in using modern JavaScript tools and techniques - 300 Level
 
-----------------
-In this demo, you will build an Office Add-in using a number of popular JavaScript tools and techniques. 
+In this demo, you will build an Office Add-in using Angular a popular JavaScript framework. 
```

Comment:
Suggest adding a comma after the word "Angular" (i.e., "...using Angular, a popular JavaScript framework.")

Reply:
Fixed.